packages/@stimulus/test/src/dom_test_case.ts

```diff
@@ -38,6 +38,16 @@ export class DOMTestCase extends TestCase {
     return event
   }
 
+  async triggerEventPassivable(selectorOrTarget: string | EventTarget, type: string, bubbles: boolean = true) {
```

Comment:
Is this method necessary just to avoid the IE <= 11 `preventDefault()` implementation in `triggerEvent()`? Maybe we can figure out how to condition it away instead. E.g. maybe the third argument to `triggerEvent()` can become an options object.